app/models/product_import/entry_processor.rb

```diff
@@ -166,6 +166,7 @@ def save_new_product(entry)
 
       product = Spree::Product.new
       product.assign_attributes(entry.attributes.except('id'))
+      product.supplier_id = entry.enterprise_id
```

Comment:
The commit message doesn't explain why this change is needed. Is it necessary because of renaming supplier to enterprise? Or is it fixing a bug?

Reply:
It's because of the renaming, yes. The product object here needs the `supplier_id` to be set.